src/bmeg/vertex.py

```diff
@@ -43,10 +43,10 @@ class Allele(Vertex):
     chromosome: str
     start: int
     end: int
-    reference_bases: str
-    alternate_bases: str
-    annotations: list = None
-    myvariantinfo: dict = None
+    reference_bases: Union[None, str]
```

Comment:
In what case would we want to allow reference_bases or alternate_bases to be None?

Reply:
g2p `features` missing ref | alt

without union for ref, alt:
```
from_source	159
is_amplification	6515
is_deletion	3128
is_fusion_acceptor	4620
is_fusion_donor	4463
is_inact_mut	197
is_loss	6895
is_mutation	2783
is_missing_location	13205
```

with Union[None,str] for ref, alt:

```
is_missing_location	13205
```

Reply:
Amplification / deletion I would think refers to a copy number alteration not an allele. The same goes for fusions - they are structural events not alleles. @kellrott what do you think?